manifests/cluster-network-addons/0.42.0/operator.yaml

```diff
@@ -155,7 +155,7 @@ spec:
         - name: MACVTAP_CNI_IMAGE
           value: quay.io/kubevirt/macvtap-cni@sha256:407f75760fc096666becfa45d94f51757ebbe8f382e9e7b57ceeded0b8cfb6b8
         - name: OPERATOR_IMAGE
-          value: quay.io/kubevirt/cluster-network-addons-operator:0.42.0
+          value: quay.io/kubevirt/cluster-network-addons-operator:v0.42.0
```

Comment:
Could you add the `v` in 0.41.0 as well please?

Reply:
see please 
https://github.com/kubevirt/cluster-network-addons-operator/pull/555#issuecomment-694166006